Processing/UIColor+ProcessingColor.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UIColor+ProcessingColor.swift
+//  Processing
+//
+//  Created by David Jensenius on 2018-02-28.
+//  Copyright © 2018 David Jensenius. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIColor {
+    class func processing() -> UIColor {
+        return UIColor(red: 32.0/255.0, green: 58.0/255.0, blue: 92.0/255.0, alpha: 1.0)
+    }
+    
+    class func batteryFull() -> UIColor {
```

Comment:
The `batteryFull()` color seems odd to me I think we don't need it (tbh I dont know why I have it in my app right now 😂 – I probably copied it from one of my other projects)

Reply:
Haha! Okay!